apps/condo/domains/meter/components/Meters/MeterPageContent.tsx

```diff
@@ -20,6 +22,7 @@ import { useMeterReadingFilters } from '@condo/domains/meter/hooks/useMeterReadi
 import { Meter, MeterPageTypes, METER_TAB_TYPES, PropertyMeter } from '@condo/domains/meter/utils/clientSchema'
 import { getMeterTitleMessage } from '@condo/domains/meter/utils/helpers'
 import { TicketPropertyField } from '@condo/domains/ticket/components/TicketId/TicketPropertyField'
+import { StatusSelect } from '@condo/domains/ticket/components/TicketStatusSelect'
```

Comment:
we need to move component outside ticket domain (in common domain) since we use it in other domain

Reply:
done